### DISCUSSION TITLE
Zamorak GWD darkens overlay as well

### DISCUSSION THREAD

**Discussion by FearrMe** [runelite/runelite#15348] [Ideas]
Stuff like xp orbs and boost information is darkened. Doubt this is supposed to happen, seeing as RS' own kc counter and the xp tracker aren't darkened.

**Comment by deathbeam** [comment]
This is due to on what layer we draw our overlays. Idk if this can be fixed in nice way.

**Comment by tr33s** [comment]
Maybe we can just remove the darkening? XD

**Comment by deathbeam** [comment]
It's part of the game, I don't think we should.

**Comment by Nightfirecat** [comment]
We're not looking at changing this behavior presently. For what it's worth, the way GPU implements drawing these darkening widgets does not darken RuneLite's overlays. It's only with the GPU plugin turned off that you will encounter this.
